jibri/rootfs/etc/cont-init.d/10-config

```diff
@@ -47,9 +47,13 @@ usermod -aG ${host_audio_group} jibri
 [ -z "${JIBRI_INSTANCE_ID}" ] && export JIBRI_INSTANCE_ID=$(date +%N)
 
 # always recreate configs
-tpl /defaults/config.json > /etc/jitsi/jibri/config.json
+tpl /defaults/jibri.conf > /etc/jitsi/jibri/jibri.conf
 tpl /defaults/logging.properties > /etc/jitsi/jibri/logging.properties
 
+# change the resolution in xorg-video-dummy file if var is not empty
+[[ ! -z "$XORG_DUMMY_VIDEO_RESOLUTION" ]] &&  \
+sed -i -e "s/Virtual 1920 1080/Virtual ${XORG_DUMMY_VIDEO_RESOLUTION}/g" /etc/jitsi/jibri/xorg-video-dummy.conf
```

Comment:
@saghul Can you please suggest a better way to configure it? Right now we have set `XORG_DUMMY_VIDEO_RESOLUTION = "1280 720"` if `JIBRI_FFMPEG_RESOLUTION = "1280x720"`.  I thought of splitting `JIBRI_FFMPEG_RESOLUTION` but it will add some extra lines of code.